javatools/build.gradle.kts

```diff
@@ -105,14 +120,15 @@ val assemble by tasks.existing {
 val sanityCheckJar by tasks.registering {
     group = VERIFICATION_GROUP
     description =
-        "If the properties are enabled, verify that the javatools.jar file is sane (contains expected packages and files), and optionally, that it has a certain number of entries."
+        "If the properties are enabled, verify that the javatools.jar file is sane " +
```

Comment:
Use a """ literal instead of plussing things together. This looks weird.

Reply:
No serious preference. For an output with constant changes or $ resolution of variables and function calls, I prefer to be "fully escaped" when doing a multi line printout. The + has been in the way before, for example when I reformat code, and it's added, then change the message or add stuff in it, and then auto reformat, and then I get another plus. Suddenly it's a combination of three word lines and thirty word lines that are plussed together. Here, I can auto format and get it adjusted without pluses, and IntelliJ tries to create equally long lines, lower than the maximum number of characters, and will word wrap nicely. That is definitely more readable. But The real question is if we need that huge description text. I did want to mention the possibility of checking entry count, though. And I do want to keep the verification mechanism, since it catches dependency bugs early when building a javatools. Accidentally missing something in javatools can trigger errors at runtime, or much later in the build, so that's why this semi-ugly pattern matcher is here in the first place. It actually started saving me serious time when I changed dependency configurations and ways to build the javatools.